outputs/openfaas.go

```diff
@@ -0,0 +1,84 @@
+package outputs
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"strconv"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/google/uuid"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+// NewOpenfaasClient returns a new output.Client for accessing Kubernetes.
+func NewOpenfaasClient(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics, statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+	if config.Openfaas.Kubeconfig != "" {
+		restConfig, err := clientcmd.BuildConfigFromFlags("", config.Openfaas.Kubeconfig)
+		if err != nil {
+			return nil, err
+		}
+		clientset, err := kubernetes.NewForConfig(restConfig)
+		if err != nil {
+			return nil, err
+		}
+		return &Client{
+			OutputType:       "OpenFaaS",
```

Comment:
```suggestion
			OutputType:       Openfaas,
```

We can use the constant

Reply:
done

Reply:
done